Probesys/bileto
# imp: Schedule the collect of emails every minute

## Related issue(s)



https://github.com/Probesys/bileto/issues/331

## Changes



- setup the Messenger async transport to use Doctrine
- add a worker in the docker-compose file
- execute the message in the MailboxController synchronously
- setup a scheduler to execute the messages every minute

## How to test manually



- update your env and start the worker by relaunching `make docker-start`
- send an email to the mailbox `support@example.com`
- check that a ticket is created after max 1 minute automatically

## Checklist
